Dockerfile

```diff
@@ -1,6 +1,33 @@
-FROM python:3.9.13-alpine3.16
+FROM ghcr.io/kbase/cdm-spark-standalone:pr-28
 
-# Install Base Packages
-RUN apk --no-cache add bash curl 
+# Switch to root to install packages
+USER root
 
-ENTRYPOINT ["sleep 1d"]
+RUN apt-get update && apt-get install -y \
+    libpq-dev gcc \
+    && rm -rf /var/lib/apt/lists/*
+
+ENV SPARK_JARS_DIR=/opt/bitnami/spark/jars
+
+ENV CONFIG_DIR=/opt/config
+COPY ./config/ ${CONFIG_DIR}
+ENV SPARK_FAIR_SCHEDULER_CONFIG=${CONFIG_DIR}/spark-fairscheduler.xml
```

Comment:
this doesn't seem to be used anywhere?

Reply:
yea. It's used in the spark session model I will include it in the next PR.